notebooks/00_datasets.ipynb

```diff
@@ -1,214 +0,0 @@
-{
```

Comment:
can we not delete the notebooks?

This repo needs to be educative and the notebooks for now are like playground to figure things out. Like rough notes.

Reply:
Adding notebooks to `.gitignore` as discussed.